### PR TITLE
<Table/> - refactor story page 

### PR DESCRIPTION
**Pull request by daugerdas**
https://jira.wixpress.com/browse/DSM-847

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-ui-tpa**

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_6836/

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  Project Name: **wix-style-react**

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6836

File: packages/wix-style-react/src/Table/docs/examples/14_TableInfiniteScrollExample.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable */
```
Comment (mykas) [inline packages/wix-style-react/src/Table/docs/examples/14_TableInfiniteScrollExample.js:1]: Change to:
```
() => {
  const initialData = [
    { firstName: 'Meghana', lastName: 'Bishop' },
    { firstName: 'Sara', lastName: 'Porter' },
    { firstName: 'Deborah', lastName: 'Rhodes' },
    { firstName: 'Walter', lastName: 'Jenning' },
  ];

  const containerRef = React.useRef(null);

  const [count, setCount] = React.useState(5);
  const [data, setData] = React.useState([]);
  const [container, setContainer] = React.useState(null);

  const simulateFetchFromServer = () =>
    setTimeout(() => {
      let data = [];
      const more = count + 5;
      for (let i = 0; i < more; i++) {
        data = data.concat(initialData);
      }
      setCount(more);
      setData(data);
    }, 1000);

  React.useEffect(() => {
    setContainer(containerRef);
    simulateFetchFromServer();
  }, []);

  const columns = [
    { title: 'First', render: row => row.firstName },
    { title: 'Last', render: row => row.lastName },
  ];

  return (
    <div ref={containerRef} style={{ maxHeight: '258px', overflow: 'auto' }}>
      <Table
        infiniteScroll
        loadMore={simulateFetchFromServer}
        itemsPerPage={20}
        hasMore={true}
        scrollElement={container && container.current}
        data={data}
        columns={columns}
      >
        <Table.Content />
      </Table>
    </div>
  );
};
```

Comment (daugerdas) [reply packages/wix-style-react/src/Table/docs/examples/14_TableInfiniteScrollExample.js:1]: changed in 9e096d1

File: packages/wix-style-react/src/Table/docs/examples/17_TableEmptyStateExample.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+
+() => {
```
Comment (mykas) [inline packages/wix-style-react/src/Table/docs/examples/17_TableEmptyStateExample.js:3]: Please follow this guideline to make code more readable: https://gist.github.com/mykas/6fa3cac9e08a5c6455ef2ce5545739e1#4-complex-structure-components-examples-can-be-daunting-to-comprehend-in-such-cases-there-is-value-in-abstracting-code-into-smaller-modules-in-order-to-reduce-mental-effort-in-understanding-the-code

File: packages/wix-style-react/src/Table/docs/examples/14_TableInfiniteScrollExample.js (hunk 1 above)
Comment (daugerdas) [reply packages/wix-style-react/src/Table/docs/examples/14_TableInfiniteScrollExample.js:1]: changed `Infinite Scroll and Bulk Selection` example similarly in bc84ab6

File: packages/wix-style-react/src/Table/docs/examples/17_TableEmptyStateExample.js (hunk 2 above)
Comment (daugerdas) [reply packages/wix-style-react/src/Table/docs/examples/17_TableEmptyStateExample.js:3]: Thank you! Updated in a450e02 and f6f4015

**Review by mykas — APPROVED** [review]

**Pull request merged**